sdk/python/kfp/client/client.py

```diff
@@ -760,6 +760,51 @@ def run_pipeline(
             IPython.display.display(IPython.display.HTML(html))
         return response.run
 
+    def archive_run(self, run_id: str):
+        """Archives a run.
+
+        Args:
+            run_id: id of the run.
+
+        Returns:
+            Object. If the method is called asynchronously, returns the request
```

Comment:
Two thoughts unrelated to your specific changes:

1. We don't enable the async functionality via the `Client`, so we probably don't want to include that in the docstrings. The existing pattern is to include, so feel free to keep it for now. If you want to update the docstrings throughout the Client class, feel free. Otherwise, let me know and I will.

2. We should also probably omit the `Object.` part of this docstring, since it doesn't tell us much. `isinstance(<almost anything>, object) == True`. Same as above: feel free to include since this is the existing pattern. If you want to update the docstrings throughout the Client class, feel free. Otherwise, let me know and I will.

Reply:
1. I went ahead and stripped all `If the method is called asynchronously, returns the request thread.` references in all of the docstrings in the `Client` class.
2. Scrapped `Returns: Object.` across the board as well.